src/helpers/settingsHelper.ts

```diff
@@ -160,4 +160,13 @@ export class SettingsHelper {
             workspaceConfiguration.update("appcenter.showStatusBar", false);
         }
     }
+
+    public static linkTwoApps(): boolean {
+        const workspaceConfiguration = vscode.workspace.getConfiguration();
+        if (workspaceConfiguration.has("appcenter.linkTwoApps")) {
```

Comment:
Move to SettingsHelper class as it seems that we use it for all communication with settings now?